lorien/InfiniteCanvas/Tools/SelectionTool.gd

```diff
@@ -36,16 +36,19 @@ func _ready():
 
 # ------------------------------------------------------------------------------------------------
 func tool_event(event: InputEvent) -> void:
-	if event.is_action_pressed("copy_strokes") || event.is_action_pressed("duplicate_strokes"):
+	var duplicate_pressed = Utils.event_pressed_bug_workaround("duplicate_strokes", event)
+	var copy_pressed = Utils.event_pressed_bug_workaround("copy_strokes", event)
+	var paste_pressed = Utils.event_pressed_bug_workaround("paste_strokes", event)
```

Comment:
Can you change these to a `:=` assignment?  (i'm always a bit nit-picky about typing😄)

Reply:
Yes - don't worry, I understand - good catch! Quickly looked if there is a linter that can pick this up, but the gdscript linters are a bit too niche to cover type hinting it seems.

Anyway - fixed now.

Reply:
Yes, there are a couple of linters but but most of them aren’t relly feature complete. Thanks!